requirements.txt

```diff
@@ -1,6 +1,7 @@
 Django==2.2
 django-authtools==1.7.0
 djangorestframework==3.12.2
-djangorestframework-simplejwt==4.6.0
+djangorestframework-simplejwt==4.4.0
```

Comment:
Why did you change the version?

Reply:
djangorestframework-simplejwt==4.6.0 version is not available anymore. 
![error](https://user-images.githubusercontent.com/32267051/100416747-665e6280-30a9-11eb-9bcc-15958df83830.png)
So I have installed version 4.4.0. Maybe that was a temporary issue. I will check it again if version 4.6.0 is critical for the project.

Reply:
I can see that you have added a lot of things coming with the packages we have in the original requirements file.
Please, let's reduce the list to the original one (and just add the swagger package).
Thank you!

Reply:
Done!